integration-tests/browser_integration_test.go

```diff
@@ -0,0 +1,168 @@
+// Package integrationtests contains broad integration integrationtests that
+// include a bootstrap node, a standalone node, and a browser node.
+package integrationtests
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/0xProject/0x-mesh/rpc"
+	"github.com/0xProject/0x-mesh/scenario"
+	"github.com/0xProject/0x-mesh/zeroex"
+	"github.com/chromedp/chromedp"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBrowserIntegration(t *testing.T) {
+	if !integrationTestsEnabled {
+		t.Skip("Integration tests are disabled. You can enable them with the --integration flag")
```

Comment:
The whole point of this `--integration` flag was that, prior to this PR, the integration tests had some additional requirements and took a lot longer to run. We didn't want to include them in `make test-all` or `make test-go` because of this.

However, in this PR there are now multiple "integration tests" that don't involve the browser and don't have any special requirements. Can we rename this flag to something like `--enable-browser-integration-tests`? You will also need to update the Makefile accordingly.